### PR TITLE
Prepare design doc ahead of balloting

### PR DESCRIPTION
**Pull request by aphillips**
- Rename the design doc.
- Cross out rejected options 2 and 5
- Add notes to 2 and 5 calling this out

(other changes may be added from the previous thread in #503 and WG call notes from 2023-10-30)

### CODE REVIEW

**Review by eemeli — APPROVED** [review]
It's fine, though two further changes could be considered:
1. The "Proposed Design" section could explicitly call out that we're balloting for which one to pick.
2. The 2 & 5 options could have their contents hidden within `<details>` blocks as already used later on in the doc.

File: exploration/delimiting-variant-patterns.md [hunk 1]
```diff
@@ -13,16 +13,62 @@ Status: **Proposed**
...
+> Using [instant runoff voting](https://en.wikipedia.org/wiki/Instant-runoff_voting), rank
+> your choice for how to handle delimiting of patterns in _complex messages_.
+
+The **_deadline_** is **1700 (5 PM) in the `America/Los_Angeles` time zone on Saturday, 4 November, 2023**
+
+- A group member in good standing MAY submit one vote consisting of a ranked set of choices
+  up until the deadline.
+- A group member MAY edit, change, or delete their vote up until the deadline.
+- Votes MUST be submitted as a comment on github issue [#505](https://github.com/unicode-org/message-format-wg/issues/505).
+  Group members who cannot submit a comment on this issue should contact the chair (@aphillips) for assitance.
```
Comment (eemeli) [inline exploration/delimiting-variant-patterns.md:46]:
```suggestion
  Group members who cannot submit a comment on this issue should contact the chair (@aphillips) for assistance.
```

File: exploration/delimiting-variant-patterns.md [hunk 2]
```diff
@@ -13,16 +13,62 @@ Status: **Proposed**
...
+
+### Definitions
+
+**_complex messages_** are messages that contain _declarations_, _selectors_, or both according
+to the ABNF.
+
+**_simple messages_** are messages consisting only of a _pattern_, with no _declarations_ or _selectors_.
+
+**_group member in good standing_** is any member of the MessageFormat mailing list or
+watcher of the message-format-wg github repo who has not be banned.
```
Comment (eemeli) [inline exploration/delimiting-variant-patterns.md:65]:
```suggestion
watcher of the message-format-wg github repo who has not been banned.
```

File: exploration/delimiting-variant-patterns.md [hunk 3]
```diff
@@ -227,11 +278,15 @@ Cons:
...
 - May be difficult for translators to add quotes when needed.
 
-### Trim All Unquoted
+### 4. Trim All Unquoted
 
-In this alternative, all non-code whitespace is trimmed
+In this alternative, non-simple patterns are trimmed
 and we do not allow/provide for pattern quoting.
-Instead, PEWS whitespace must be individually quoted.
+Instead, pattern-sigificant whitespace must be individually quoted.
```
Comment (eemeli) [inline exploration/delimiting-variant-patterns.md:285]:
```suggestion
Instead, pattern-significant whitespace must be individually quoted.
```

File: exploration/delimiting-variant-patterns.md [hunk 4]
```diff
@@ -227,11 +278,15 @@ Cons:
...
+Note that only the first character of whitespace has to be quoted
+or that an empty literal can be used to start or end the pattern.
```
Comment (eemeli) [inline exploration/delimiting-variant-patterns.md:287]:
```suggestion
Note that only the first character of whitespace has to be quoted,
or an empty literal `{||}` can be used to start or end the pattern.
```

File: exploration/delimiting-variant-patterns.md [hunk 5]
```diff
@@ -249,20 +304,24 @@ Instead, PEWS whitespace must be individually quoted.
...
+> is part of the pattern but all of the whitespace after the period at
+> the end of this sentence is included in the pattern.
 >
 >       {||}
 >```
 
 Pros:
 - Code is special, whitespace is not.
-- Makes PEWS into a "special event", alerting developers to the non-I18N aspects of it?
+- Makes whitespace into a "special event", alerting developers to the non-I18N aspects of it?
```
Comment (eemeli) [inline exploration/delimiting-variant-patterns.md:316]:
```suggestion
- Makes leading and trailing whitespace into a "special event",
  alerting developers to the non-I18N aspects of it?
```

File: exploration/delimiting-variant-patterns.md [hunk 6]
```diff
@@ -175,13 +221,16 @@ Pros:
 - The quoting reduces the number of in-pattern escapes to the open/close sequence.
   and the placeholder sequence sigils.
 - Since the pattern is already quoted, translators never have to add pattern quotes
-  in order to add PEWs to a given pattern.
+  in order to include whitespace into a given pattern.
```
Comment (eemeli) [inline exploration/delimiting-variant-patterns.md:224]:
```suggestion
  in order to add leading or trailing whitespace in a given pattern.
```

**Review by eemeli — APPROVED** [review]
See line comments for minor fixes.

File: exploration/delimiting-variant-patterns.md (hunk 6 above)
Comment (aphillips) [reply exploration/delimiting-variant-patterns.md:224]: Mostly good, but "include"/"into" I think is slightly clearer.

```suggestion
in order to include leading or trailing whitespace into a given pattern.
```

**Comment by eemeli** [comment]
The last paragraph of the introductory "Summary" section seems rather opinionated. Could it be dropped?

> With the current syntax, the boundary between pattern and the remainder of the message is always clear
because the pattern is either the entire string
or enclosed in the `{{...}}` quoting required by the syntax.

**Comment by echeran** [comment]
> The last paragraph of the introductory "Summary" section seems rather opinionated. Could it be dropped?
> 
> > With the current syntax, the boundary between pattern and the remainder of the message is always clear
> > because the pattern is either the entire string
> > or enclosed in the `{{...}}` quoting required by the syntax.

It seems pretty straightforward and objective to me. What's opinionated about it?

**Review by echeran — APPROVED** [review]
LGTM

**Pull request merged**